### PR TITLE
Fix updating domain for scales

### PR DESCRIPTION
**Pull request by zBritva**
Change layout of scale editor and labels. Set defaults for data axis.

### CODE REVIEW

File: src/core/prototypes/plot_segments/axis.ts [hunk 1]
```diff
@@ -1242,7 +1242,7 @@ export function buildAxisWidgets(
         m.vertical(
           m.horizontal(
             [0, 1],
-            m.label("Auto range min value"),
+            m.label("Auto min value"),
```
Comment (danmarshall) [inline src/core/prototypes/plot_segments/axis.ts:1245]: strings file

File: src/core/prototypes/scales/linear.ts [hunk 2]
```diff
@@ -145,41 +149,39 @@ export class LinearScale extends ScaleClass<
       manager.sectionHeader("Domain"),
       manager.row("Start", manager.inputNumber({ property: "domainMin" })),
       manager.row("End", manager.inputNumber({ property: "domainMax" })),
-      manager.sectionHeader("Range"),
-      manager.mappingEditor("Start", "rangeMin", { defaultValue: 0 }),
-      manager.mappingEditor("End", "rangeMax", { defaultAuto: true }),
-      manager.sectionHeader("Scale export properties"),
+      // manager.sectionHeader(""),
+      manager.row("", manager.label("Auto update values")),
```
Comment (danmarshall) [inline src/core/prototypes/scales/linear.ts:153]: strings

**Review by danmarshall — APPROVED** [review]

File: src/core/prototypes/scales/linear.ts (hunk 2 above)
Comment (zBritva) [reply src/core/prototypes/scales/linear.ts:153]: https://github.com/microsoft/charticulator/pull/493

File: src/core/prototypes/plot_segments/axis.ts (hunk 1 above)
Comment (zBritva) [reply src/core/prototypes/plot_segments/axis.ts:1245]: https://github.com/microsoft/charticulator/pull/493

**Pull request merged**